metadata_service/api/popular_tables.py

```diff
@@ -27,6 +27,6 @@ def __init__(self) -> None:
         self.client = get_proxy_client()
 
     def get(self) -> Iterable[Union[Mapping, int, None]]:
-        limit = request.args.get('limit', 10)
+        limit = request.args.get('limit', 10, type=int)
```

Comment:
thanks. is this line change needed?

Reply:
Without the type the `limit` variable would come in as a string instead of an integer. The function interface says it wants an integer. I believe the downstream function will work either way (the value is just being put into a query). 

I can remove it if you would prefer.

Reply:
I think it is fine :) Thanks for the info.